Linking with Gazebo in pr2_gazebo causes a build error in another package

Using `find_package` to gazebo in https://github.com/PR2/pr2_simulator/blob/kinetic-devel/pr2_gazebo/CMakeLists.txt#L4 makes the following build error in `pr2eus`, which uses `find_package` to `pr2_gazebo` in https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/CMakeLists.txt#L50
```
CMake Error at /opt/ros/kinetic/share/pr2_gazebo/cmake/pr2_gazeboConfig.cmake:148 (message):
  Project 'pr2eus' tried to find library '-lpthread'.  The library is neither
  a target nor built/installed properly.  Did you compile project
  'pr2_gazebo'? Did you find_package() it before the subdirectory containing
  its code is included?
Call Stack (most recent call first):
  CMakeLists.txt:50 (find_package)
```
The similar problem is also reported in https://github.com/ros-planning/navigation/issues/612 and https://github.com/ros/catkin/issues/856

I built master branch of `pr2eus` on kinetic and got a build error as follows:
```
$ source /opt/ros/kinetic/setup.bash
$ mkdir -p test_ws/src && cd test_ws/src
$ git clone https://github.com/jsk-ros-pkg/jsk_pr2eus.git
$ cd ..
$ catkin build
-----------------------------------------------------------
Profile:                     default
Extending:             [env] /opt/ros/kinetic
Workspace:                   /home/pazeshun/test_ws
-----------------------------------------------------------
Source Space:       [exists] /home/pazeshun/test_ws/src
Log Space:         [missing] /home/pazeshun/test_ws/logs
Build Space:        [exists] /home/pazeshun/test_ws/build
Devel Space:        [exists] /home/pazeshun/test_ws/devel
Install Space:      [unused] /home/pazeshun/test_ws/install
DESTDIR:            [unused] None
-----------------------------------------------------------
Devel Space Layout:          linked
Install Space Layout:        None
-----------------------------------------------------------
Additional CMake Args:       None
Additional Make Args:        None
Additional catkin Make Args: None
Internal Make Job Server:    True
Cache Job Environments:      False
-----------------------------------------------------------
Whitelisted Packages:        None
Blacklisted Packages:        None
-----------------------------------------------------------
Workspace configuration appears valid.

NOTE: Forcing CMake to run for each package.
-----------------------------------------------------------
[build] Found '4' packages in 0.0 seconds.                                                          
[build] Updating package table.                                                                     
Starting  >>> catkin_tools_prebuild                                                                 
Finished  <<< catkin_tools_prebuild                [ 1.4 seconds ]                                  
Starting  >>> pr2eus                                                                                
____________________________________________________________________________________________________
Errors     << pr2eus:cmake /home/pazeshun/test_ws/logs/pr2eus/build.cmake.000.log                   
-- set EUSDIR to /opt/ros/kinetic/share/euslisp/jskeus/eus
-- set ARCHDIR to Linux64
CMake Error at /opt/ros/kinetic/share/pr2_gazebo/cmake/pr2_gazeboConfig.cmake:148 (message):
  Project 'pr2eus' tried to find library '-lpthread'.  The library is neither
  a target nor built/installed properly.  Did you compile project
  'pr2_gazebo'? Did you find_package() it before the subdirectory containing
  its code is included?
Call Stack (most recent call first):
  CMakeLists.txt:50 (find_package)


cd /home/pazeshun/test_ws/build/pr2eus; catkin build --get-env pr2eus | catkin env -si  /usr/bin/cmake /home/pazeshun/test_ws/src/jsk_pr2eus/pr2eus --no-warn-unused-cli -DCATKIN_DEVEL_PREFIX=/home/pazeshun/test_ws/devel/.private/pr2eus -DCMAKE_INSTALL_PREFIX=/home/pazeshun/test_ws/install; cd -
....................................................................................................
Failed     << pr2eus:cmake                         [ Exited with code 1 ]                           
Failed    <<< pr2eus                               [ 1.7 seconds ]                                  
Abandoned <<< pr2eus_moveit                        [ Unrelated job failed ]                         
Abandoned <<< pr2eus_tutorials                     [ Unrelated job failed ]                         
[build] Summary: 1 of 4 packages succeeded.                                                         
[build]   Ignored:   1 packages were skipped or are blacklisted.                                    
[build]   Warnings:  None.                                                                          
[build]   Abandoned: 2 packages were abandoned.                                                     
[build]   Failed:    1 packages failed.                                                             
[build] Runtime: 3.1 seconds total.                                                                 
[build] Note: Workspace packages have changed, please re-source setup files to use them.
```

--- a/pr2_gazebo/CMakeLists.txt
+++ b/pr2_gazebo/CMakeLists.txt
@@ -1,17 +1,9 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(pr2_gazebo)
 
-find_package(gazebo REQUIRED)
-find_package(PkgConfig REQUIRED)
-pkg_check_modules(SDF sdformat REQUIRED)
-
 find_package(catkin REQUIRED)
 
-catkin_package(
-  DEPENDS 
-    GAZEBO
-    SDF
-)
+catkin_package()
 
 # Install Gazebo Scripts
 install(

